code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -427,10 +427,13 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	new /obj/item/weapon/pen(src)
 	pdachoice = isnull(H) ? 1 : (ishuman(H) ? H.pdachoice : 1)
 	switch(pdachoice)
-		if(1) icon = 'icons/obj/pda.dmi'
-		if(2) icon = 'icons/obj/pda_slim.dmi'
-		if(3) icon = 'icons/obj/pda_old.dmi'
-		if(4) icon = 'icons/obj/pda_rugged.dmi'
+	//ARFS edit START - for ceciPDAs

```

Comment:
Make a copy of this proc into a new override file. You don't need to edit or delete this one at all. In the new override remove declarations for what it is though, i.e. since this is a proc you don't need to declare it as a proc again, because the proc already exists, you're just writing over what it does.